package.json

```diff
@@ -94,6 +94,7 @@
 		"@types/firefox-webext-browser": "^82.0.0",
 		"@types/jsdom": "^16.2.10",
 		"@types/mini-css-extract-plugin": "^1.4.3",
+		"@types/node": "^16.0.0",
```

Comment:
This shouldn't be necessary, but… https://github.com/xojs/xo/issues/552#issuecomment-873406603